### ISSUE TITLE
Submission dates are being displayed in UTC

### ISSUE THREAD

**Issue by dleard** [bcgov/cas-registration#880]
The submission date for a BORO ID request is being displayed in UTC. It should be displayed in PST.

This opens up a larger discussion on timestamps. Should we be saving our timestamps in PST in the database (Openshift by default will use UTC and we'll have to write a wrapper like we did in CIIP to convert our timestamps OR potentially we could set a localization setting for postgres) or do we save all timestamps in UTC in the database and convert as needed on the frontend. (Metabase also has a localization setting that will convert these for us automatically)

The AC here is simply that the submission date should be displayed in PST, but the above question affects the implementation. Marking this one Attn: Developer until an agreement has been reached,

**Comment by pbastia** [comment]
@dleard  I think hosting a localization setting for postgres opens an entire can of worms (do we do that for all timestamps? just this one? how do we interpret any datetime object that doesn't have a tz information?), and that we should keep any date in the database a UTC one to remove all ambiguity.
Converting to PST can be an API (probably preferred so the users' localization is less likely to interfere) or frontend concern

**Comment by DataVillage** [comment]
I agree that timestamps should be stored in UTC.   Storing in PST can add other complications like having to handle daylight savings time.   Also in addition to submission dates, I'm thinking timestamps could be used for all sorts of things like event tracking in logs, date and time of logins, last updates etc.  There also may be situations when the system user is not in BC (for example the head office of an organization may be located in a different time zone).  To ensure consistency, in all of these cases, using UTC is likely more straightforward and less likely to cause confusion.  To ensure user friendliness, the API can handle time conversions to PST as needed.

**Comment by dleard** [comment]
Thank you for your input, we will leave timestamps as UTC in the database. This means this ticket can be done after release.

**Comment by BCerki** [comment]
This was already completed at some point: 

![Image](https://github.com/bcgov/cas-registration/assets/77306764/f281cffc-5eee-4437-8f36-f9acb404dacd)

Moving to PO column
